packages/ui-extensions/src/surfaces/checkout/components/Badge/Badge.ts

```diff
@@ -0,0 +1,31 @@
+import {createRemoteComponent} from '@remote-ui/core';
+
+import type {Size, VisibilityProps} from '../shared';
+
+type Tone = 'default' | 'critical' | 'subdued';
+
+export interface BadgeProps extends VisibilityProps {
+  /**
+   * The tone of the badge being rendered. Indicates its level of importance,
+   * where subdued provides the least emphasis, while critical indicates the highest level of urgency.
+   *
+   * @default 'default'
+   */
+  tone?: Tone;
+  /**
+   * The size of the badge being rendered.
+   *
+   * @default 'base'
+   */
+  size?: Extract<Size, 'base' | 'small'>;
+  /**
+   * A label that describes the purpose or contents of the element. When set,
+   * it will be passed as `aria-label` to underlying element and announced to buyers using assistive technologies.
+   */
```

Comment:
Can you use the description from here:
https://github.com/Shopify/ui-api-design/blob/9d608f8b909c0dd2d4275b40b1ee16a56e93f03b/components/Badge/Badge.ts#L24-L28